server.js

```diff
@@ -9,11 +9,25 @@
 
 require('babel-core/register');
 
+var optional = require("optional");
```

Comment:
I don't really like the optional thing. What about you always have a message.js file where the implementation is something like:

```javascript

module.exports = function(reviewers, defaultMessageGenerator, buildMentionSentence) {

  // If you want to customize the message, put your own custom logic here

  return defaultMessageGenerator(reviewers);
}